File: Triangulo.java
public class Triangulo extends Figuras{
    double base;
    double altura;
    double lado1;
    double lado2;
    double lado3;

    public Triangulo(double base, double altura, double lado1, double lado2, double lado3){
        this.altura = altura;
        this.base = base;
        this.lado1 = lado1;
        this.lado2 = lado2;
        this.lado3 = lado3;
    }
    
    @Override
    double area() {
        return(base * altura)/2;
    }

    @Override
    double perimetro() {
        return lado1 + lado2 + lado3;
      
    }
}
    

File: Triangulo.py
from Figuras import Figuras

class Triangulo(Figuras):
    def __init__(self, base, altura,lado1,lado2,lado3):
        self.base = base
        self.altura = altura
        self.lado1 = lado1
        self.lado2 = lado2
        self.lado3 = lado3
    def area(self):
        return (self.base * self.altura)/2
    def perimetro(self):
        return self.lado1 + self.lado2 + self.lado3
    def __str__(self): 
        return "Triangulo de base: " + str(self.base) + " y altura: " + str(self.altura) + " y lados: " + str(self.lado1) + ", " + str(self.lado2) + ", " + str(self.lado3)
    